.npmrc

```diff
@@ -1 +1 @@
-registry=http://registry.npmjs.org/
+package-lock=false
```

Comment:
This file can be deleted, since `package-lock.json` is ignored

Reply:
It can't. There's a bug in npm that breaks switching branches with a `package-lock.json` present, because the `package.json` changes without the `package-lock.json` changing. The recommend workaround is `package-lock=false`. Ignoring the lock is optional, but I figured we might as well to prevent noise before the next `npm install`.

Reply:
Sounds reasonable. I think we should probably keep the registry entry though. Sometimes people mistakenly have a registry configured globally and it causes a bunch of trouble for them. Thoughts?

Reply:
I'm not sure. I removed it mainly because I noticed this is HTTP and not HTTPS, so the default should be HTTPS. Some users might also prefer a registry that's back compatible with npm's like GitHub Packages.

Reply:
Yeah, I guess I'm find with this change 👍